401codechallenges/src/main/java/code401challenges/hashtable/Hashtable.java

```diff
@@ -91,42 +87,42 @@ public boolean add(String key, String value) {
     // **************** Get Method ***************
 
     // Returns value for a key
-    public String get(String key) {
+    public Value get(Key key) {
 
         int index = hash(key);
+        if (bucketArray[hashCode()] == null) {

```

Comment:
you do not use your index on line 93, replace hashCode() with index

Reply:
Thank you! Refactored.